docs/collaborations.md

```diff
@@ -346,6 +350,7 @@ Get Pending Collaborations
 A collection of all the user's pending collaborations can be retrieved with
 [`collaborations.getPending(callback)`](http://opensource.box.com/box-node-sdk/jsdoc/Collaborations.html#getPending).
 
+<!-- sample get_collaborations -->
```

Comment:
Since this is a pretty specific case, we might want to make this one a variant, e.g. `get_collaborations pending`

Reply:
Not sure. I think that endpoint only returns pending collaborations.